docs/polymer-modulize-howto.md

```diff
@@ -3,7 +3,7 @@
 1. Edit `dependency-map.json` and bump every `^3.0.0-pre.<N>` version to
    `^3.0.0-pre.<N+1>`.
 
-2. Run `modulizer --clean --npm-version 3.0.0-pre.<N+1> --repo $(cat <modulizer dir>/docs/polymer-modulize-repos.txt | tr '\n' ' ') --delete-files bower.json '**/*.d.ts' types/`. This should take 2-3 minutes.
+2. Run `node --max-old-space-size=8192 ./bin/modulizer.js --clean --npm-version 3.0.0-pre.<N+1> --repo $(cat <modulizer dir>/docs/polymer-modulize-repos.txt | tr '\n' ' ') --delete-files bower.json '**/*.d.ts' types/`. This should take 2-3 minutes.
```

Comment:
Less disruptive to use an environment variable so the rest of the command doesn't need to change: `NODE_OPTIONS="--max_old_space_size=8192" modulizer ...`

Reply:
TIL!